v0/destinations/ga4/transform.js

```diff
@@ -63,13 +91,10 @@ const responseBuilder = (message, { Config }) => {
     case "gtag":
       // gtag.js uses client_id
       // GA4 uses it as an identifier to distinguish site visitors.
-      rawPayload.client_id =
-        getDestinationExternalID(message, "ga4ClientId") ||
-        get(message, "anonymousId") ||
-        get(message, "messageId");
+      rawPayload.client_id = getGA4ClientId(message, Config);
       if (!isDefinedAndNotNull(rawPayload.client_id)) {
         throw new CustomError(
-          "ga4ClientId, anonymousId or messageId must be provided",
+          `[Google Analytics 4]: ${Config.clientIdFieldIdentifier}, ga4ClientId, anonymousId or messageId must be provided`,
```

Comment:
If the custoem is not setting `config.clientIdFieldIdentifier` it will show undefined. Can we use some generic term here?

Reply:
@ItsSudip i already discussed this with @akashrpo , this error is just for transformer test cases because in real this error will never be thrown because at last we are depending on `messageId` or `anonymousId`. transformer will never throw this error since we wanted to cover all the test cases that's this error is there (also this error is suggested by @akashrpo )